src/styles/style.css

```diff
@@ -0,0 +1,115 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font-family: 'DM Sans', sans-serif;
+}
+
+.container {
+  margin: auto 10%;
+  background: linear-gradient(90deg, #b88009 0%, #22c1c3 100%);
+  padding: 0 2%;
+}
+
+#game-board {
+  display: flex;
+  gap: 30%;
+  margin-top: 30px;
+}
+
+h1 {
+  text-align: center;
+  margin-top: 50px;
+  font-size: 50px;
+  font-weight: bolder;
+  color: rgb(235, 235, 235);
+  padding-top: 50px;
+}
```

Comment:
- [ ] Also, keep your design similar to the wireframe. The input form should be on the right.


### Your design
![Screenshot from 2022-07-30 03-26-00](https://user-images.githubusercontent.com/26767383/181848125-9c1d1b63-7c32-42f1-b93e-4e96825076ab.png)

### Given design
![Screenshot from 2022-07-30 03-24-45](https://user-images.githubusercontent.com/26767383/181848136-4d35e933-0de3-4c5d-b1fc-addc9221d03b.png)